manifests/05_rbac.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aws-ebs-csi-driver-operator-role
+subjects:
+  - kind: ServiceAccount
+    name: aws-ebs-csi-driver-operator
+    namespace: openshift-aws-ebs-csi-driver-operator
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Are we not going to bother with fine grained permissions for operator? It is okay with me.. but I just want to ensure that we are using same policy in CSV as here.

Reply:
We should bother! I'm just taking some shortcuts to get e2e.